code/src/java/pcgen/persistence/lst/StatsAndChecksLoader.java

```diff
@@ -101,7 +101,7 @@ else if ("ALIGNMENTNAME".equals(key))
 			return null;
 		}
 		String name = token.substring(colonLoc + 1);
-		if ((name == null) || (name.isEmpty()))
+		if (name.isEmpty())
```

Comment:
I've noticed we use certain conventions because of bad actors in the past, will this cause any issues?